frontends/mit-open/src/pages/FieldPage/FieldPage.tsx

```diff
@@ -32,6 +33,9 @@ const FieldPage: React.FC = () => {
     channelType && (
       <FieldPageSkeleton name={name} channelType={channelType}>
         <p>{fieldQuery.data?.public_description}</p>
+        <TestimonialCard
+          offerors={channelType === "offeror" ? [name] : undefined}
+        />
```

Comment:
I believe that only offeror ("unit") pages should have Testimonial. Other channel pages, like "Departments" do not.

Figma for Department Detail pages:
<img width="616" alt="Screenshot 2024-06-11 at 1 19 20 PM" src="https://github.com/mitodl/mit-open/assets/9010790/31b4ebaf-9df8-4ef7-a60a-def151918ad9">

But this is showing all testimonials for department pages (and topic pages).

<img width="1178" alt="Screenshot 2024-06-11 at 1 20 10 PM" src="https://github.com/mitodl/mit-open/assets/9010790/306b5839-3334-4d88-9694-43ffc3e7e9ec">